lib/cli.js

```diff
@@ -270,6 +270,20 @@ List of characters to copy, belongs to previously declared "--font". Examples:
     help: 'Pad bitmap line endings to whole bytes.'
   });
 
+  parser.add_argument('--stride', {
+    choices: [ 4, 8, 16, 32, 64 ],
```

Comment:
Add 
- `0` for backward compatibility. Meaning e.g. with bpp=4 lines won't be byte aligned
- `1` to align e.g. bpp=4 at least to byte boundary. 

```suggestion
    choices: [0, 1, 4, 8, 16, 32, 64 ],
```